<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComposeWindow</class>
 <widget class="QDialog" name="ComposeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="line_to">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labe_to">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TO*:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_title">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TITLE*:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_text">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TEXT*:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="box_text">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>301</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_attachment">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>431</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_attachment">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>380</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Attachment:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>SEND</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_eroare2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_eroare1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
